app/Models/Annotation.php

```diff
@@ -313,6 +313,10 @@ public function toAnnotatorArray($userId = null)
         $item['flags'] = $this->flags();
         $item['seen'] = $this->seen;
 
+        //fix for #804: Time stamps for annotations are off by several hours
+        //convert date to RFC2822 string
+        $item['created_at'] = date_format(date_create($item['created_at']), "r");
+
```

Comment:
@dkd903 This is great, but you'll need to do this for both `created_at` and `updated_at` for consistency across the site.  Were you able to determine why `toRFC2822String()` above was mangling the timezone info?